### PR TITLE
Add a Jetstream intro

### PR DESCRIPTION
**Pull request by tdsmith**
Closes https://github.com/mozilla/jetstream/issues/537.

### CODE REVIEW

File: docs/jetstream/jetstream.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+id: jetstream
+title: Jetstream
```
Comment (LZoog) [inline docs/jetstream/jetstream.md:3]: What do you think about calling this "about Jetstream" or "Jetstream overview" just so the sidebar isn't redundant on Jetstream > Jetstream?

**Review by LZoog — APPROVED** [review]
LGTM, have one question/suggestion about the doc title.

You may want to get someone else that can better vet the info you provided here to r+ before merge though. 🙂

File: docs/jetstream/jetstream.md (hunk 1 above)
Comment (tdsmith) [reply docs/jetstream/jetstream.md]: sgtm!

**Comment by tdsmith** [comment]
Thanks for the review. The content's not new so I don't think it needs another look for that.

**Pull request merged**